tools/gradle-plugin/src/test/java/io/smallrye/openapi/gradleplugin/SmallryeOpenApiPluginTest.java

```diff
@@ -71,7 +71,7 @@ void taskPropertiesInheritance() {
         ext.applicationPathDisable.set(true);
         ext.configProperties.set(new File("/foo/bar"));
         ext.encoding.set("ISO-8859-1");
-        ext.openApiVersion.set("3.0.0");
+        ext.openApiVersion.set("3.1.0");
```

Comment:
It might be good to keep this different from the default. Maybe `3.1.1` or something. Even if it's not valid, we can test that the configuration is actually taking effect.

Reply:
If we're going to do a non-valid one, can I suggest something like 999.999.999. That way is unambiguously invalid.

Reply:
That's fine with me, thanks.